$OpenBSD: patch-i3status_conf,v 1.15 2017/01/29 15:12:42 jasper Exp $

Adjust config file to be reasonably useful on OpenBSD out of the box.
NB: _first_ keyword for network interfaces doesn't work yet.

--- i3status.conf.orig	Sun Jan 29 13:01:35 2017
+++ i3status.conf	Sun Jan 29 16:07:58 2017
@@ -7,6 +7,7 @@
 # If the above line is not correctly displayed, fix your editor first!
 
 general {
+	output_format = "i3bar"
         colors = true
         interval = 5
 }
@@ -18,20 +19,31 @@ order += "ethernet _first_"
 order += "battery all"
 order += "load"
 order += "tztime local"
+order += "cpu_temperature cpu0"
+order += "cpu_temperature acpitz0"
+order += "volume master"
 
+cpu_temperature cpu0 {
+         format = "C: %degrees C"
+         path = "cpu0"
+}
+
+cpu_temperature acpitz0 {
+        format = "TZ: %degrees C"
+}
+
 wireless _first_ {
         format_up = "W: (%quality at %essid) %ip"
         format_down = "W: down"
 }
 
 ethernet _first_ {
-        # if you use %speed, i3status requires root privileges
         format_up = "E: %ip (%speed)"
         format_down = "E: down"
 }
 
 battery all {
-        format = "%status %percentage %remaining"
+        format = "%status %percentage \% %remaining"
 }
 
 tztime local {
@@ -44,4 +56,8 @@ load {
 
 disk "/" {
         format = "%avail"
-}
+}
+
+volume master {
+	format = "vol: %volume"
+}
\ No newline at end of file
